packages/tocco-ui/src/EditableValue/utils.js

```diff
@@ -53,3 +53,21 @@ export const momentJStoToFlatpickrFormat = format => {
   _forOwn(formattingMap, (v, k) => { format = format.replace(v, k) })
   return format
 }
+
+/*
+ * Returns format separators provided by intl API.
+ */
+export const parseLocalePlaceholder = countryCode => {
+  const thousandSeparator = (1111).toLocaleString(countryCode).replace(/1/g, '')
+  const decimalSeparator = (1.1).toLocaleString(countryCode).replace(/1/g, '')
+
+  return {thousandSeparator, decimalSeparator}
+}
+
+/*
+ * Limits formatted value of react-number-format
```

Comment:
ah ok havent seen that this function is very react-number-format specific. utils is not the right place then. move it back to Component.